### PR TITLE
Add links to report

### LINKED ISSUE #2202: Pool detail page: add direct links to reports + fix reviewer avatar
## Background
Highlighting the financial reporting that Centrifuge app is capable of and making it easier for the user to find the new reports. Squeeze in one bug fix for the pool analysis section. 

## Spec
- [ ] Add direct links to the reports in the reporting tab
-`Balance sheet`
-`Profit & Loss report`
-`Cashflow statement`
-`View all `

Image
Image

### Bug
- [ ] Fix bug to display pool reviewer avatar 

Image

### LINKED ISSUE #2219: Bug: Issuer details shown twice
## Bug report
- On the pool detail page, the issuer information section is shown twice.

## Fix
- Remove issuer details from the structure section 

## How to reproduce
- Go to https://app.centrifuge.io/pools/4139607887 

Screenshot 2024-06-18 at 12 58 32

### PR DESCRIPTION
**Pull request by kattylucy**
### Description

- Add direct links to the reports in the reporting tab
- Fix duplicate issuer details 

#2202 
#2219 

### Approvals

- [ ] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### CODE REVIEW

**Review by sophialittlejohn — COMMENTED** [review]
Nice work 🔥 

One small thing: Reports/Pool analysis is optional when creating a pool, so you'll just need to remove the card entirely if it doesn't exist. The rest of the cards (Structure) should then slide up to fill the space, like in the screenshot. You may need to refactor the Grid items a bit to make each Card independent
Screenshot 2024-06-18 at 17 39 45

**Comment by hieronx** [comment]
@sophialittlejohn @kattylucy  reports are not optional, they always exist for all pools. Only the Pool Analysis is optional. So I would anyways keep the card, only hiding the Pool Analysis section if that is not set.

**Comment by kattylucy** [comment]
Screenshot 2024-06-19 at 7 07 36 AM

Screenshot 2024-06-19 at 7 17 22 AM


Please refer to image for expected behavior

File: centrifuge-app/src/components/IssuerSection.tsx [hunk 1]
```diff
@@ -2,46 +2,78 @@ import { PoolMetadata } from '@centrifuge/centrifuge-js'
...
-            {report.author.title}
+    <Shelf flexDirection="column" alignItems="flex-start">
+      <Shelf marginBottom={20}>
+        <LabelValueStack
+          label=""
+          value={
+            <Text variant="body3">
+              <Shelf flexWrap="wrap" gap={2} alignItems="flex-start">
+                {reportLinks.map((link, i) => (
+                  <RouterTextLink to={`${pathname}/reporting${link.href}`} key={`${link.label}-${i}`}>
```
Comment (hieronx) [inline centrifuge-app/src/components/IssuerSection.tsx:36]: In the design, they were slightly larger and the underline was different, I think like this
```css
font-size: 16px;
text-decoration: none;
border-bottom: 1px solid #000;
```

We probably want to add a new component for this.

See https://www.figma.com/design/xzA29JHIc0TTWvUhKrKwcF/Reporting?node-id=3501-2260&t=2SrSgXYI6fna2zh7-4

File: centrifuge-app/src/components/IssuerSection.tsx [hunk 2]
```diff
@@ -2,46 +2,78 @@ import { PoolMetadata } from '@centrifuge/centrifuge-js'
...
-              src={cent.metadata.parseMetadataUrl(report.author.avatar.uri)}
-              alt=""
-            />
-          )}
-          <Text variant="body2">
-            Reviewer: {report.author.name}
-            <br />
-            {report.author.title}
+    <Shelf flexDirection="column" alignItems="flex-start">
+      <Shelf marginBottom={20}>
```
Comment (hieronx) [inline centrifuge-app/src/components/IssuerSection.tsx:29]: Also would make this 
```suggestion
      <Shelf marginBottom={30}>
```

File: centrifuge-app/src/components/IssuerSection.tsx (hunk 1 above)
Comment (onnovisser) [reply centrifuge-app/src/components/IssuerSection.tsx:36]: The TextLink inherits the font size. Changing the Text variant to `body1` should do it

**Comment by onnovisser** [comment]
The issuer details card is now not showing for Tinlake pools. Anna suggested making it it's own card in the linked ticket https://github.com/centrifuge/apps/issues/2219

File: centrifuge-app/src/components/IssuerSection.tsx (hunk 1 above)
Comment (kattylucy) [reply centrifuge-app/src/components/IssuerSection.tsx:36]: I don't have access to the figma files, do you mind sharing those with me? @hieronx TY!

Comment (hieronx) [reply centrifuge-app/src/components/IssuerSection.tsx:36]: Approved @kattylucy

Comment (kattylucy) [reply centrifuge-app/src/components/IssuerSection.tsx:36]: Also Im using the same component as links down on the issuer details - I was aiming for consistency but should I go ahead and change it anyways? 

Screenshot 2024-06-20 at 7 43 52 AM

Comment (hieronx) [reply centrifuge-app/src/components/IssuerSection.tsx:36]: Yeah these links are more important so should have larger font sizes etc.

Comment (kattylucy) [reply centrifuge-app/src/components/IssuerSection.tsx:36]: Will do. Thanks both!

File: centrifuge-app/src/components/IssuerSection.tsx [hunk 3]
```diff
@@ -2,46 +2,78 @@ import { PoolMetadata } from '@centrifuge/centrifuge-js'
...
-            {report.author.title}
+    <Shelf flexDirection="column" alignItems="flex-start">
+      <Shelf marginBottom={30}>
+        <LabelValueStack
+          label=""
+          value={
+            <Text variant="body3">
+              <Shelf flexWrap="wrap" gap={2} alignItems="flex-start">
+                {reportLinks.map((link, i) => (
+                  <RouterTextLink variant="heading" to={`${pathname}/reporting${link.href}`} key={`${link.label}-${i}`}>
```
Comment (onnovisser) [inline centrifuge-app/src/components/IssuerSection.tsx:36]:
```suggestion
            <Text variant="body1">
              <Shelf flexWrap="wrap" gap={2} alignItems="flex-start">
                {reportLinks.map((link, i) => (
                  <RouterTextLink to={`${pathname}/reporting${link.href}`} key={`${link.label}-${i}`}>
```

**Comment by kattylucy** [comment]
Ok hoping this covers all the scenarios 🤞🏾 

Screenshot 2024-06-20 at 9 32 24 AM
Screenshot 2024-06-20 at 9 33 34 AM
Screenshot 2024-06-20 at 9 48 20 AM

**Review by sophialittlejohn — APPROVED** [review]
🔥

**Pull request merged**